app.py

```diff
@@ -1587,12 +1660,14 @@ def export_matched_trips():
             download_name="matched_trips.gpx",
         )
 
+
 @app.route("/api/export/streets")
 def export_streets():
     location = request.args.get("location")
     export_format = request.args.get("format")
 
-    streets_data, _ = generate_geojson_osm(json.loads(location), streets_only=True)
+    streets_data, _ = generate_geojson_osm(
+        json.loads(location), streets_only=True)
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider handling ignored error return value</b></div>
<div id="fix">

Consider improving code readability by using more descriptive variable names for the second returned value from `generate_geojson_osm()` since it's being ignored with `_`
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<blockquote>Check the AI-generated fix before applying</blockquote>
<div id="code">


```suggestion
    streets_data, error = generate_geojson_osm(
        json.loads(location), streets_only=True)
    if error:
        return jsonify({'error': error}), 400

```

</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/realronaldrump/everystreet-new/pull/94#issuecomment-2543406597>#91e3ec</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged